README.md

```diff
@@ -126,7 +126,7 @@ To build the job from source and run against a local cassandra, in Spark's stand
 
 ```bash
 # Build the spark jobs
-$ ./mvnw -DskipTests clean install
+$ mvn -DskipTests clean install
```

Comment:
we don't publish the maven wrapper in our source dist. this README ends up in the source dist. we have gotten nitpicked about references to mvnw in previous releases. this tries to avoid that nit picking